### ISSUE TITLE
RabbitMQ backend

### ISSUE THREAD

**Issue by olegstepura** [Flared/dramatiq-abort#9]
Hi! Is there any plans to add RabbitMQ backend?

**Comment by isra17** [comment]
Not really, a key/value store is needed. You could always hack something with RabbitMQ, but this is not planned.
